packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -110,8 +110,20 @@ class ScrollBehavior {
   /// {@macro flutter.gestures.monodrag.DragGestureRecognizer.multitouchDragStrategy}
   ///
   /// By default, [MultitouchDragStrategy.latestPointer] is configured to
-  /// create drag gestures for all platforms.
-  MultitouchDragStrategy get multitouchDragStrategy => MultitouchDragStrategy.latestPointer;
+  /// create drag gestures for non-Apple platforms, and
+  /// [MultitouchDragStrategy.maxAllPointers] for Apple platforms.
+  MultitouchDragStrategy getMultitouchDragStrategy(BuildContext context) {
+    switch (getPlatform(context)) {
+      case TargetPlatform.macOS:
+      case TargetPlatform.iOS:
+        return MultitouchDragStrategy.maxAllPointers;
+      case TargetPlatform.linux:
+      case TargetPlatform.windows:
+      case TargetPlatform.android:
+      case TargetPlatform.fuchsia:
```

Comment:
Maybe?
```suggestion
      default:
```

Reply:
Is it idiomatic to have a default or to just enumerate all the cases and let the compiler check?

Reply:
Consistent with elsewhere, an compilation error is thrown if forget to specify behavior for a new platform.